storybook/stories/Markdown.js

```diff
@@ -4,6 +4,7 @@ import { ScrollView, StyleSheet, View } from 'react-native';
 import { storiesOf } from '@storybook/react-native';
```

Comment:
Can you try to migrate to ts?
I'm not sure it's going to work though, because we might not have the prop lint.